tests/Endpoints/MeshTreeTest.php

```diff
@@ -21,16 +21,6 @@ protected function getFixtures(): array
         ];
     }
 
-    /**
-     * @inheritDoc
-     */
-    public function readOnlyPropertiesToTest()
```

Comment:
the base-method for this data provider function was was moved into the `tests/Endpoints/PutEndpointTestable.php`.
however, since all MeSH-tests are read-only by nature, testing for PUT never happens, so i yanked these orphaned methods without replacement from all MeSH endpoint tests.